pmda/test/test_rmsd.py

```diff
@@ -6,16 +6,14 @@
 # (see the file AUTHORS for the full list of names)
 #
 # Released under the GNU Public Licence, v2 or any higher version
-from __future__ import absolute_import, division, print_function
 
+from __future__ import absolute_import, division, print_function
 import pytest
 from numpy.testing import (assert_almost_equal, assert_array_equal,
                            assert_array_almost_equal)
-
 import MDAnalysis
 from MDAnalysisTests.datafiles import (PSF, DCD)
-
-from pmda import rms
```

Comment:
This line should still work, together with `rms.RMSD()`, right?

I  generally like imports that keep a name space attached to objects – makes it a lot easier to see where stuff comes from. I am ok with using `from pmda.rms import RMSD` here, just want to confirm that `rms.RMSD` will remain to work.

Reply:
Yes, you can still do 

`from pmda import rms`
`ca = universe.select_atoms("name CA")`
`universe.trajectory.rewind()`
`rmsd1 = rms.RMSD(ca, ca).run()`